src/types.ts

```diff
@@ -144,6 +149,16 @@ export interface ServerOptions {
   };
 }
 
+export type NodeHttpHandlerFunc = (
```

Comment:
This is a duplicate of the `NodeHttpHandler` type from [line 319](https://github.com/OskarLebuda/srvx/blob/6491d322590485fea5fbd64b7058ccdf09b51f0e/src/types.ts#L319) of this file.